modules/pel/peltool/peltool.py

```diff
@@ -289,6 +365,10 @@ def main():
                         help='Used with -d, only look for files with this extension (e.g. ".pel")')
     parser.add_argument('-x', '--delete', dest='delete', action='store_true',
                         help='Delete original file after parsing')
+    if inBMC:
+        parser.add_argument('-l', '--list',
+                            help='List PELs',
+                            action='store_true')
```

Comment:
I think I've come around to having a -l in the nonBMC case, since now it's as easy as:

```
else:
           parser.add_argument('-l', '--list',
                            help='List PELs',
                            dest=list_directory')
  
          # if not isdir(list_dir) fail ...
```
does that seem like it would work?

Reply:
Added the functionality.